model.py

```diff
@@ -119,6 +122,11 @@ def main():
                 auc_roc = roc_auc_score(all_labels, all_probs)
                 precision, recall, _ = precision_recall_curve(all_labels, all_probs)
                 auc_pr = auc(recall, precision)
+
+                # Example: Log metrics to the database
+                model_type = "binary classification"  # Adjust based on your specific model type
+                log_training_metrics(auc_pr, auc_roc, loss.item(), model_type)
```

Comment:
Oops, you've moved this method into the utils.py file, but forgot to import it at the top of the file. Please do so and test running pipeline.py to verify that everything works. You can then discard any changes to trained_model and the .onnx files before committing and re-requesting review.

Also, update requirements.txt to include any new library imports you've added in this PR.

For the future, I would recommend working in a branch of universal-gestures-unity instead of your own fork: it makes it easier for us to test and review your code changes :)

Reply:
I implemented the requested changes.